ThunderCloud/ContentController.swift

```diff
@@ -661,86 +675,95 @@ public class ContentController: NSObject {
             return false
         }
         
+        os_log("Loading manifest.json as dictionary", log: self.contentControllerLog, type: .debug)
         guard let manifest = manifestJSON as? [String: Any] else {
             
-            os_log("Can't cast manifest as dictionary", log: self.contentControllerLog, type: .error)
+            os_log("Can't cast manifest as dictionary\n %@", log: self.contentControllerLog, type: .error, ContentControllerError.invalidManifest.localizedDescription)
 
             callProgressHandlers(with: .verifying, error: ContentControllerError.invalidManifest)
             return false
         }
         
-        
         if (!self.fileExistsInBundle(file: "app.json")) {
             
-            os_log("app.json is missing", log: self.contentControllerLog, type: .error)
+            os_log("%@", log: self.contentControllerLog, type: .error, ContentControllerError.missingAppJSON.localizedDescription)
 
             callProgressHandlers(with: .verifying, error: ContentControllerError.missingAppJSON)
             return false
         }
+        os_log("app.json exists", log: self.contentControllerLog, type: .debug)
         
         if (!self.fileExistsInBundle(file: "manifest.json")) {
             
-            os_log("manifest.json is missing", log: self.contentControllerLog, type: .error)
+            os_log("%@", log: self.contentControllerLog, type: .error, ContentControllerError.missingManifestJSON.localizedDescription)
 
             callProgressHandlers(with: .verifying, error: ContentControllerError.missingManifestJSON)
             return false
         }
+        os_log("manifest.json exists", log: self.contentControllerLog, type: .debug)
         
         // Verify pages
+        os_log("Verifying pages", log: self.contentControllerLog, type: .debug)
         guard let pages = manifest["pages"] as? [[String: Any]] else {
             
-            os_log("No pages in manifest", log: self.contentControllerLog, type: .error)
-            callProgressHandlers(with: .verifying, error: ContentControllerError.invalidManifest)
+            os_log("%@", log: self.contentControllerLog, type: .error, ContentControllerError.missingPages.localizedDescription)
+            callProgressHandlers(with: .verifying, error: ContentControllerError.missingPages)
```

Comment:
Should this be called `missingPages` or `manifestMissingPages`? `missingPages` could be a bit confusing seeing as it could also refer to pages missing from the actual bundle 😊

Reply:
Good point. Will rename.